spec/models/converters/date_time_converter_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe DateTimeConverter do
+  subject { described_class }
+
+  it 'converts strings to dates' do
+    date_string = '2020-01-01T12:05:02+08:00'
+    expect(described_class.convert(date_string)).to eq(DateTime.parse(date_string))
```

Comment:
unless DateTime is preferable in DateTimeConverter, this should also be `Time.zone.parse`

Reply:
Yes, you're right, I don't think this is necessary.

I think I can just remove this converter and let rails handle it and we'll be fine. For the data we have (which is just test data in a format we came up with since we have zero idea of what will eventually be provided), I think we're safe with that.